### PR TITLE
New Prosec stamp and minor Justice loadout adjustments

### PR DESCRIPTION
**Pull request by Tomce795**
## About the PR
Added new stamps for Prosecutor. Also adjusted their round-spawn equipment so they start with the stamp and with a Space Law book (also added one for Clerk),

## Why / Balance
There are specific documents that can only be approved by certain members of Justice. Prosecutor can approve things like warrants and they typically write out paperwork for prosecuting during a trial, But Prosec only has access to the Lawyer stamp, like Attorney. This should help differentiate who actually stamped what and miiiight encourage more Prosec to do more paperwork.

## Technical details
Added a Prosecutor stamp, stamp component and appropriate prototype.
Adjusted Prosecutor round start equipment to spawn with the stamp.
Added a Space Law book to both Prosecutor and Clerk on round start.

## Media
Stamp example

## Requirements
- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
Probably none?

**Changelog**
:cl:
- add: Added a new stamp for Prosecutor and added it to their starting loadout. Also added the Space Law book to the starting loadout for both Prosec and Clerk.

### CODE REVIEW

**Comment by Tomce795** [comment]
Unironically unsure if the new sprites are supposed to be in here or the Delta-V folder which has the mantis one. I'm sure one of you's will yell at me thou :blunt:

**Comment by Avalon-Proto** [comment]
I feel like I should mention this. The clerk already has a stamp which is called the Notary Stamp. So the clerk stamp here is irrelevant

**Comment by Tomce795** [comment]
Guh. Where does this stamp spawn? In a Clerk locker or something?

**Comment by Avalon-Proto** [comment]
Yep, it does

**Comment by Tomce795** [comment]
Okay general idea is to then to remove the Clerk stamp, make the prosec one the only one added. Also probably shift about the stamp to the Delta-V resource folder (but the notary stamped paper sprite isn't there for some reason?). Do Prosec get their own locker? If not, I'll keep the loadout changes so that they spawn with it.

**Comment by Radezolid** [comment]
Prosec doesn't get a locker, gotta add it to their loadout.

**Comment by Tomce795** [comment]
I dunno why the upstream files are still... modified, but that's to figure out later or to webedit away :blunt:

I believe the other stuff is okay, so nap time for me.

**Comment by Unkn0wnGh0st333** [comment]
Uhh change the 🆑 to only be one bullet, don't need two different change logs for the same PR (unless it's a massive PR) :blunt:

File: Resources/Locale/en-US/paper/stamp-component.ftl
Comment (MilonPL) [inline Resources/Locale/en-US/paper/stamp-component.ftl:1]: untouch this file

File: Resources/Prototypes/Entities/Objects/Misc/rubber_stamp.yml
Comment (MilonPL) [inline Resources/Prototypes/Entities/Objects/Misc/rubber_stamp.yml:1]: untouch

**Review by MilonPL — CHANGES_REQUESTED** [review]

**Comment by Tomce795** [comment]
Untouched upstream files. `git checkout upstream/master` to the rescue. GitHub is wacky and confuzzling but I think this is now good to merge after tests.

**Review by MilonPL — APPROVED** [review]

**Pull request merged**